packages/status-react/src/routes/index.tsx

```diff
@@ -23,6 +23,20 @@ interface Props extends Config {
   meta?: string
 }
 
+// todo: use a better way to handle this
+const RootGate = (props: { children: JSX.Element }) => {
+  const { client } = useProtocol()
+
+  // todo!: use sorted chats
```

Comment:
follow-up: discuss: for example, move `useSortedChats` logic to `status-js` and get the sorted chats from `community._chats`